main_routers/vrm_router.py

```
@@ -0,0 +1,212 @@
+                    "path": str(vrm_file),
+                    "url": f"{VRM_USER_PATH}/{vrm_file.name}",  
+                    "type": "vrm",
+                    "size": vrm_file.stat().st_size if vrm_file.exists() else 0,
+                    "location": "user"  
+                })
+
+        return JSONResponse(content={
+            "success": True,
+            "models": models
+        })
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)